src/lib/section.svelte

```diff
@@ -0,0 +1,34 @@
+<script>
+  export let isVisible = false;
+  export let title;
+</script>
+
+<div>
+  <h6 class="subtitle is-6 has-background-link-light is-unselectable pl-1">
+    <button
+      on:click={() => {
+        isVisible = !isVisible;
+      }}
+      class="is-flex is-justify-content-space-between">
+      <span>{title}</span>
+      <ion-icon name={isVisible ? "chevron-down-outline" : "chevron-up-outline"} />
+    </button>
+  </h6>
+  <div style:display={isVisible ? null : "none"}>
+    <slot />
+  </div>
+</div>
+
+<style>
+  button {
+    /* reset */
+    padding: unset;
+    border: unset;
+    outline: unset;
+    font: unset;
+    color: unset;
+    background: unset;
```

Comment:
This gets rid of the default styling of the button (though, it should probably have focus (and possibly hover) styles added back in.

Reply:
Agree that this would be a good idea